dap-etl/extraction/src/test/scala/org/broadinstitute/monster/dap/cslb/CslbExtractionPipelineSpec.scala

```diff
@@ -25,15 +25,6 @@ class CslbExtractionPipelineSpec extends AnyFlatSpec with Matchers {
     cslbArms shouldBe List("annual_2020_arm_1", "annual_2021_arm_1", "annual_2022_arm_1")
   }
 
-  it should "use current date as the endTime when none is provided" in {
```

Comment:
❓ Why delete this test?

Reply:
I was using the CSLB tests in this file as a template for the new AFUS tests.
When modifying this specific test, I realized that the test logic would not hold up over time. If this test were run a few years into the future, it would expect a much longer list of arms than what is currently hardcoded in the test logic. This means that we would have to update this test every year to make sure we are including the newest arm that would be included.